infra/base-images/base-builder/Dockerfile

```diff
@@ -15,6 +15,7 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-clang
+RUN apt-get update
```

Comment:
Not sure why I needed this but `docker build -t gcr.io/oss-fuzz-base/base-builder infra/base-images/base-builder` did not work without it